app/assets/stylesheets/application.scss

```diff
@@ -241,3 +237,14 @@ a {
   color: $foot;
   padding: 50px;
 }
+
+.back-ground-text {
+  background-color: rgba(65, 92, 108, 0.5);
+  padding: 10px;
```

Comment:
Properties should be ordered background-color, margin-left, margin-right, padding